src/Core/x/xModel.h

```diff
@@ -1,10 +1,67 @@
 #ifndef XMODEL_H
 #define XMODEL_H
 
+#include <rwcore.h>
+#include <rpworld.h>
+
+#include "xAnim.h"
+#include "xLightKit.h"
+#include "xSurface.h"
 #include "xMath3.h"
 
+struct xModelInstance;
+
+// TODO: this probably can be simplified
+struct _class
+{
+	xVec3* verts;
+};
```

Comment:
The name `_class` is usually generated for unnamed structs. I would define this struct as part of the declaration for `xModelInstance::anim_coll`, like this.

```
struct xModelInstance
{
    /* ... */
    struct
    {
        xVec3* verts;
    } anim_coll;
};
```